packages/rn-tester/js/components/RNTesterTheme.js

```diff
@@ -44,19 +49,25 @@ export const RNTesterLightTheme = {
   SecondaryLabelColor: '#3c3c4399',
   TertiaryLabelColor: '#3c3c434c',
   QuaternaryLabelColor: '#3c3c432d',
+
   PlaceholderTextColor: '#3c3c434c',
+
   SystemBackgroundColor: '#ffffffff',
```

Comment:
TIL We've been hardcoding these colors inRNTester :(
We being React Native in general

Reply:
Yeah, it's unfortunate :-(